bcda/monitoring/monitoring.go

```diff
@@ -26,6 +27,27 @@ func (a apm) Start(msg string, w http.ResponseWriter, r *http.Request) *newrelic
 	return nil
 }
 
+func (a apm) NewTransaction(name string, ctx context.Context) (*newrelic.Transaction, context.Context) {
+	if a.App != nil {
+		txn := a.App.StartTransaction(name)          // transaction trace
+		newRelicCtx := newrelic.NewContext(ctx, txn) // parent context
+		return txn, newRelicCtx
```

Comment:
do we need to return the `txn` variable? it looks like in the [example for the integration](https://github.com/newrelic/go-agent/blob/b704e94bdd0cdd982464fa7ae55e2cb5e8e51f0a/v3/integrations/nrpgx/example/sql_compat/main.go#L34), we can just return the context and use that in the query to the database. 
```
txn := app.StartTransaction("postgresQuery")
ctx := newrelic.NewContext(context.Background(), txn)
row := db.QueryRowContext(ctx, "SELECT count(*) FROM pg_catalog.pg_tables")
```

Reply:
[these docs](https://pkg.go.dev/github.com/newrelic/go-agent/v3/integrations/nrpgx) also tell us that the above would require us to add the integration in during the database setup, as well: 
```db, err := sql.Open("nrpgx", "host=localhost port=5432 user=postgres dbname=postgres password=docker sslmode=disable")```

Reply:
I would also be curious if a new relic transaction is meant to hold a specific segment (like a database call) or if we should be creating a transaction earlier when a request comes in.

Reply:
that makes sense to me! there is already a helper function that starts a transaction for [web requests here](https://github.com/CMSgov/bcda-app/blob/2ab83d60a3bb007661a7fd2fad23c547096a9e3c/bcda/monitoring/monitoring.go#L19).